types/react-dynamic-number/index.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for react-dynamic-number 1.7
+// Project: https://github.com/uhlryk/react-dynamic-number
+// Definitions by: Eugene Rodin <https://github.com/eugrdn>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.6
+
+import * as React from 'react';
+
+/**
+ * remove Diff & Omit when in will be placed in TS from scratch
+ */
+type Diff<T extends string, U extends string> = ({[P in T]: P} & {[P in U]: never} & {[x: string]: never})[T];
+type Omit<T, K extends keyof T> = {[P in Diff<keyof T, K>]: T[P]};
+
+interface OmittedFields {
```

Comment:
This is only used for its keys -- just use `"ref" | "value" | "onChange" | "placeholder"` instead

Reply:
done